h5p-bildetema-words-grid-view/src/components/WordAudio/WordAudio.tsx

```diff
@@ -29,6 +31,11 @@ export const WordAudio: React.FC<WordAudioProps> = ({ word, textVisible }) => {
       audioElement.pause();
       audioElement.currentTime = 0;
     } else {
+      if (contextAudioRef?.current) {
+        contextAudioRef?.current?.pause();
+        contextAudioRef!.current!.currentTime = 0;
```

Comment:
Are these null checks necessary?

```suggestion
        contextAudioRef.current.pause();
        contextAudioRef.current.currentTime = 0;
```

Reply:
You are correct! Removed in all files in newest commit.